pages/gitandrepositories.md

```diff
@@ -143,7 +143,8 @@ For more info on differences of merging vs. rebasing (and when to use which one)
 ## Summary of steps
 1. Clone your git \<username\> repository and open your terminal window to it by using ``cd``
 
-#### Note that for steps 2 and 3, developers should do this every time they begin to work. This way we as a team can minimize data loss, and can save you some time. You want to make sure that you are always up to date with the github.
+NOTE: [for steps 2 and 3, developers should do this every time they begin to work. This way we as a team can minimize data loss, and can save you some time. You want to make sure that you are always up to date with the github.]
```

Comment:
take out '[' and ']'. Its already a note, it dose not need to be parenthesized. Capitalize the first word in the sentence. change github to GitHub.

Be careful when you are changing a sentence that you are also naturally fixing the grammar.